q2_quality_control/assets/index.html

```diff
@@ -0,0 +1,156 @@
+{% extends 'base.html' %}
+
+{% block title %}q2-quality-control : {{ running_title }}{% endblock %}
+
+{% block content %}
+
+<div class="row">
+  <h1>{{ title }}</h1>
+  <div class="text-center">
+    {% if score_plot %}
+    <h1>Per-Level Accuracy</h1>
+      <a href="score_plot.pdf">
+        <img src="score_plot.png">
+        <br>
+        <p>Download as PDF</p>
+      </a>
+    {% endif %}
+    <div class="col-lg-12">
```

Comment:
Your columns (eg `col-lg-12`) should be children of semantic `row`s.

Reply:
pls elaborate. Are you saying that preceding each column I should have another `<div class="row">`?

Reply:
Can do! Functionally, [bootstrap grids](https://getbootstrap.com/docs/3.3/css/#grid) follow this pattern: a container holds rows, and rows hold columns. The `class="container"` comes from the `{% block content %}` parent template, so you don't need to do anything there, otherwise:

```jinja
{% block content %}
<h1>{{ title }}</h1>

<div class="text-center">

  <div class="row">
    <div class="col-lg-12">
      ... results ...
    </div>
  </div>

  {% if composition_regression %}
  <div class="row">
    <div class="col-lg-12">
      <h1>Linear regression between observed and expected abundances</h1>
      <a href="composition_regression.pdf">
        <img src="composition_regression.png">
        <br>
        <p>Download as PDF</p>
      </a>
    </div>
  </div>
  {% endif %}

  ... etc ...

</div>
{% endblock %}
```